i18n/en-US.properties

```diff
@@ -222,6 +222,8 @@ be.previousFile = Previous File
 be.previousSegment = Previous Segment
 # Label for print action
 be.print = Print
+# If a user has been deleted, we call the user "a prior collaborator" - meaning someone who used to be able to collaborate on the content.
+be.priorCollaborator = A Prior Collaborator
```

Comment:
Hmm not sure how this got here.

Reply:
Should go away if you rebase, there was a missed i18n build